clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -290,19 +289,16 @@ private void doSend(ClientRequest clientRequest, boolean isInternalRequest, long
             // information itself.  It is also the case when discoverBrokerVersions is set to false.
             if (versionInfo == null) {
                 if ((!discoverBrokerVersions) && (log.isTraceEnabled()))
-                    log.trace("No version information found when sending message of type {} to node {}",
-                            clientRequest.apiKey(), nodeId);
+                    log.trace("No version information found when sending message of type {} to node {}.  " +
```

Comment:
nit: We use a single space after a period throughout the rest of the code. It would be good to stay consistent.